cmd/run.go

```diff
@@ -753,158 +750,158 @@ func RunServer(ctx context.Context, config *Config) error {
 // bindRunFlags binds the cobra cmd flags to the equivalent config value being managed
 // by viper. This bridges the config between cobra flags and viper flags.
 func bindRunFlags(cmd *cobra.Command) {
-
```

Comment:
What do you think about putting this function into `util.go`? Seems a bit out of place here.

Reply:
Yeah, that's a good idea. We can colocate it next to the `MustBindPFlag` and `MustBindEnv` command utilities. 👍

Reply:
Done!